database/layer/admin-client.js

```diff
@@ -3,15 +3,44 @@ const pool = new PGPool()
 
 module.exports = {
   insertClient,
+  getClient,
   updateClientByClientId,
   deleteClientByClientId
 }
 
-async function insertClient (clientId, clientname, clientdescription, secret) {
-  const sql = 'INSERT INTO usher.clients (client_id, name, description, secret) VALUES ($1, $2, $3, $4)'
+/**
+ *
+ * Insert a new Client into database
+ *
+ * @param {string} tenantName Unique tenant to associate Client to
+ * @param {string} clientId The unique id for the new Client to create
+ * @param {string} clientName Client Name
+ * @param {string} clientDescription Optional description for the Client
+ * @param {string} secret
+ * @returns Record with the newly created object
+ */
+async function insertClient (tenantName, clientId, clientName, clientDescription, secret) {
```

Comment:
Using an object instead of multiple arguments probably makes the function easier to use.

Reply:
Can we have code styling / formatting changes like this be a totally separate PR from new API functionality?

Reply:
Yes, good idea.